table.go

```diff
@@ -117,20 +120,20 @@ func DiffEvents(t1, t2 Table, h func(e *Event)) error {
 
 	// Validate both tables have the key columns.
 	// Build lookups key columns.
-	key1Map := make(map[string]struct{}, len(key2))
+	key1Set := make(map[string]struct{}, key2Len)
```

Comment:
It's not a big deal, but my original suggestion was that writing "key 1 length" or "key 2 length" implies that it matters which one you're referring to. I thought it would be more clear if there were just one variable with the key length, which could be set after the equality assertion.

Reply:
Ah, yes of course! Good point. I will update.

Reply:
..and it is a big deal. I misinterpreted what you said and this is more clear.